### PR TITLE
Add a new public key

### PR DESCRIPTION
**Pull request by deovaliandro**
pub   ed25519/B905A377C78E929C 2021-07-24 [SC]
      90263F5E34B5D26F841B2D70B905A377C78E929C
uid                 [  full  ] Deo Valiandro. M (Wibu) <deo@gnuweeb.org>
uid                 [  full  ] Deo <deo@gnuweeb.org>
sub   cv25519/7F997A7D409F53B7 2021-07-24 [E]

Signed-off-by: deovaliandro <deo@gnuweeb.org>

### CODE REVIEW

File: keys/90263F5E34B5D26F841B2D70B905A377C78E929C.asc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+
+xjMEYPuc/RYJKwYBBAHaRw8BAQdA2f29j7PbuAHzJ/qce4wHGxOa/kY+JhYqk972GJZkvPPNFURl
```
Comment (ammarfaizi2) [inline keys/90263F5E34B5D26F841B2D70B905A377C78E929C.asc:8]: Please add the key info here as well.

**Review by ammarfaizi2 — CHANGES_REQUESTED** [review]
Please add the key info in the beginning of the line.

File: keys/90263F5E34B5D26F841B2D70B905A377C78E929C.asc (hunk 1 above)
Comment (deovaliandro) [reply keys/90263F5E34B5D26F841B2D70B905A377C78E929C.asc:8]: fix: https://github.com/GNUWeeb/pgpkeys/pull/6/commits/2a41258be7f71d5fe07d7d919c81a54ffa22d3db

**Review by ammarfaizi2 — APPROVED** [review]
Good, I am merging this.

**Pull request merged**